lib/utils/functionInfoUtils.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
```

Comment:
Those utils are AWS specific. We could wrap them in an `aws` property or rename the file to make this more obvious.